src/test/java/uk/gov/companieshouse/model/validator/RelationshipValidatorTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.companieshouse.model.validator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.companieshouse.TestData;
+import uk.gov.companieshouse.TestUtil;
+import uk.gov.companieshouse.model.Appeal;
+import uk.gov.companieshouse.model.CreatedBy;
+import uk.gov.companieshouse.model.Reason;
+
+@ExtendWith(SpringExtension.class)
+public class RelationshipValidatorTest {
```

Comment:
Well done! Could you please remove the 'public' modifier, to resolve sonar issue with the code smell

Reply:
Approved but this smell should be removed ✔️